internal/api.go

```diff
@@ -156,6 +156,12 @@ func (c *APIClient) Post(url, contentType string, body io.ReadSeeker) (resp *htt
 	// transfer encoding.
 	if rb, ok := body.(*RingBuffer); ok {
 		req.ContentLength = int64(rb.Len())
+		// Fix the content-type header to avoid chunked transfer encoding
+		// which is not supported by healthchecks.io.
```

Comment:
This is confusing and entirely correct. If the reader `healthchecks.io` as the hosted service, then it's wrong. Service at API `hc-ping.com` has no problems with chunked encoding. If one parses this as the project, Healthchecks with project homepage `healthchecks.io`, it is also by association correct, because of its dependency on Django's web server implementation.

Drop this mention.